src/appsignal/config.py

```diff
@@ -300,8 +300,6 @@ def _validate(self) -> None:
         push_api_key = self.option("push_api_key") or ""
         if len(push_api_key.strip()) > 0:
             self.valid = True
-        else:
-            print("appsignal: Push API key not set after loading config. Not starting.")
```

Comment:
Hm, not sure about this. I think it's useful to inform the user if this happens at their application's run-time. (And I think it's preferable to have a redundant message be emitted on the demo, than it is to not emit it on run-time at all)

Reply:
I think it's useful to add, and I copied it from the Ruby gem that does it, but it adds it to its logger, not always print it.

Other integrations don't do this. So until we have a better way to log it, I want to remove it.